created messages app

255AM opened the pull request:
#Created the messages page. 

#It's part of the MVP for the current project 

#Includes a data provider to get and add chats. A list function to handle the render and a card file to pass each individual message into.  Each chat sent can be see by all users. The chat includes the userName of the user who sent it, the time sent, and message sent.  Each time a message is sent or the page is loaded, the page will auto scroll to presetn the most recent messsage to the user

## Anything Else? A fun game to play is to open my app, then open slack in another window. Now try to guess which is slack and which is not. More styling is needed

MandyC6420 commented:
**Nutshell QA Review of**:  messages ticket

**Description of bug found**: 

1.  After a message is created and sent, the message text input field displays the same message which can be sent again and again until the text box is deleted and a new message is written

**Console Logs**(if any):

**Steps to reproduce defect**: Create a message, press send message, see message input field still filled with previous message.

**Expected Vs.  Actual results**: 
Expected:  Once message is sent, the text input field would be cleared to prepare for the next message to be sent

Actual results:  The message text field remains filled with the original message after the message is sent

255AM commented:
Corrected bug that caused text area to remain populated after user submits message. 
Corrected by setting state to object with empty string after submit

MandyC6420 commented:
👍 all corrections complete, wonderful job!!
